qiita_pet/handlers/study_handlers/description_handlers.py

```diff
@@ -423,6 +423,63 @@ def add_prep_template(self, study, user, callback):
 
         callback((msg, msg_level, 'prep_template_tab', pt_id, None))
 
+    def update_prep_template(self, study, user, callback):
```

Comment:
@antgonza I noticed that when a new sample/prep template is added, the uploaded file is removed from the upload folder. However, when we update the values this doesn't happen. Is this desired? IMO we should remove the file from the upload folder, so we can keep the FS cleaner (otherwise for every update that needs to be done the file is still in there...).

Another advantage of this is that if the user wants to update a value, it will download our current template, instead of using his/her old template, which I think is better. Does this make sense?